src/node_options.cc

```diff
@@ -44,6 +44,24 @@ void DebugOptions::CheckOptions(std::vector<std::string>* errors) {
     errors->push_back("[DEP0062]: `node --inspect --debug-brk` is deprecated. "
                       "Please use `node --inspect-brk` instead.");
   }
+
+  if (inspect_publish_uid_string.empty())
+    inspect_publish_uid_string = "stderr,http,binding";
```

Comment:
Can you make this the default value in `node_options.h`? That way an empty string can be passed intentionally

Reply:
done.